src/build/build.ts

```diff
@@ -115,7 +115,7 @@ export async function build(options: BuildOptions, config: ProjectConfig): Promi
 
   // Once the unbundled build stream is complete, create a service worker for the build
   let unbundledPostProcessing = Promise.all([loadSWConfig, waitFor(unbundledPhase)]).then((results) => {
-    let swConfig: SWConfig = results[0];
+    let swConfig: SWConfig|undefined = results[0] || undefined;
```

Comment:
Hm. Oh, I'm using a prerelease version of the compiler, I should switch back to stable.